containers/datalab/content/run.sh

```diff
@@ -115,6 +116,17 @@ watch_and_restart_tunnel() {
   setup_tunnel "${project_id}" "${zone}" "${instance}" "${ssh_user}"
 }
 
+check_tmp_directory() {
+    echo "Verifying that the /tmp directory is writable"
+    test_temp_file=$(mktemp --tmpdir=/tmp)
+    if [ ! -e "${test_temp_file}" ]; then
+	echo "Unable to write to the /tmp directory"
+	exit "${ERR_TMP_NOT_WRITABLE}"
```

Comment:
Just to double check, after the exit here the whole thing will be retried, right?